programs/voter-stake-registry/src/instructions/accelerate_vesting.rs

```diff
@@ -0,0 +1,47 @@
+use crate::error::*;
+use crate::state::*;
+use anchor_lang::prelude::*;
+
+#[derive(Accounts)]
+pub struct AccelerateVesting<'info> {
+    pub registrar: AccountLoader<'info, Registrar>,
+    // checking the PDA address it just an extra precaution,
+    // the other constraints must be exhaustive
+    #[account(
+      mut,
+      seeds = [registrar.key().as_ref(), b"voter".as_ref(), voter_authority.key().as_ref()],
+      bump = voter.load()?.voter_bump,
+      has_one = voter_authority,
+      has_one = registrar)]
+    pub voter: AccountLoader<'info, Voter>,
+    pub voter_authority: Signer<'info>,
```

Comment:
Hmm, thinking about this in a proposal context, it'd mean that this instruction can only go through it the voter themself agrees and executes the proposal. Is that what you intended?

It might be reasonable to allow the grant_authority to do this without voter input.

Reply:
I was thinking, similar to how Grant proposals must be executed by the voter_authority, so too should an unlock. 

(Not a tax expert) but there could potentially be tax implications of unlocking some asset of value. I believe receiving an asset that you have no ability to sell may not be taxable (e.g. a locked grant), but as you soon as you unlock and gain the ability to transfer, sell, etc. then that could be taxable. 

So I thought it might be safest for voters to have to execute.

Reply:
Right, makes sense